dockerfiles/pr-check/Dockerfile

```diff
@@ -16,11 +16,15 @@ RUN yum update --assumeyes \
         pcp \
         bzip2 \
         golang \
+        gcc-c++ \
```

Comment:
Could you please elaborate why this is needed?

Reply:
It's required for `npm`, It's been in the dependencies to install certain tools.